fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentMapper.java

```diff
@@ -67,9 +87,96 @@ public FluentClient map(CodeModel codeModel, Client client) {
             fluentClient.getExamples().addAll(examples);
         }
 
+        // live tests
+        fluentClient.getLiveTests().addAll(client.getLiveTests().stream().map(liveTests -> mapLiveTests(liveTests, fluentClient)).collect(Collectors.toList()));
+
         return fluentClient;
     }
 
+    private FluentLiveTests mapLiveTests(LiveTests liveTests, FluentClient fluentClient) {
+
+        FluentLiveTests result = new FluentLiveTests();
+        result.setClassName(liveTests.getFilename() + "Tests");
+
+        parseLiveTests: for (LiveTestCase liveTestCase : liveTests.getTestCases()) {
```

Comment:
Labelled continue usually is not a good sign. Consider to break some inner processing to methods.

Reply:
Replaced with `stream.filter`